deployments/kubernetes/install/helm/istio-cni/templates/istio-cni.yaml

```diff
@@ -144,3 +144,20 @@ kind: ServiceAccount
 metadata:
   name: istio-cni
   namespace: {{ .Release.Namespace }}
+
+---
+{{- if hasKey .Values "psp_cluster_role" }}
```

Comment:
Instead of `hasKey`, could you instead test for string emptyness, and add this into the `values*.yaml` files for documentation? The `README.md` file doesn't ship into the Istio release, so that documentation is lost. Adding that into `values*.yaml` with a comment would be very useful.

Reply:
I do !